lib/cocoapods/downloader/cache.rb

```diff
@@ -327,13 +327,15 @@ def copy_source_and_clean(source, destination, spec)
       # @return [Void]
       #
       def copy_files(files, source, destination)
-        files = files.select { |f| File.exist?(f) }
+        files = files
```

Comment:
nit. do we need to map first and then select or just select and convert to Pathname within the `select`?

Reply:
I guess we do because we use them below as pathnames